### PR TITLE
CD-60841 Makefile and Dockerfile for SAND

### PR DESCRIPTION
**Pull request by johnwu96822**
The commands for building an image is:
```
make dist
docker build -t sand -f Dockerfile-sand .
```

- [x] @johnny-lai 
- [x] @abdollar 
- [x] @tjackiw

### CODE REVIEW

**Comment by abdollar** [comment]
@johnwu96822 you can probably get rid of the vendor experiment environment variable.

**Comment by abdollar** [comment]
👍

**Comment by tjackiw** [comment]
👍  after addressing Abdul's comments

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+GLIDE = $(GOPATH)/bin/glide
+
+SRCROOT ?= $(realpath .)
+BUILD_ROOT ?= $(SRCROOT)
+
+# These are paths used in the docker image
+SRCROOT_D = /go/src/github.com/ory-am/hydra
+BUILD_ROOT_D = $(SRCROOT_D)/tmp/dist
+
+VERSION = $$(git name-rev --tags --name-only $$(git rev-parse HEAD))
```
Comment (johnny-lai) [inline Makefile]: Seems odd to be getting the version from the tag. I would think it makes more sense for the VERSION to be hard-coded. For example, this is what is done in the gems.

But I understand that this is what is done in some of our other microservices.

**Comment by johnny-lai** [comment]
lgtm.
* I would like to mention that I think getting the VERSION from the tag is an odd thing to do.
* Note that glide updates may break this build.

**Comment by johnwu96822** [comment]
@johnny-lai Removed VERSION and COMMIT, since Sand (Hydra) doesn't use it anyway...

**Comment by johnwu96822** [comment]
[lockit v3.0.2] Merged via Lockit CLI on 2016-11-14 16:04:36.466295996 -0800 PST

**Pull request merged**